update_release.sh

```diff
@@ -0,0 +1,41 @@
+name: Update Releases
+
+on:
+  push:
+    branches:
+      - main  # Trigger on pushes to the main branch (adjust as necessary)
+
+jobs:
+  update-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get the latest release tag
+        id: get_latest_release
+        run: |
+          latest_tag=$(gh release list)
+          echo "::set-output name=latest_tag::$latest_tag"
+
+      - name: Get changed files since latest release
+        id: get_changes
+        run: |
+          latest_tag=${{ steps.get_latest_release.outputs.latest_tag }}
+          git fetch origin $latest_tag --depth=1
+          git diff --name-only $latest_tag HEAD > changed_files.txt
+          echo "::set-output name=changed_files::$(cat changed_files.txt | tr '\n' ' ')"
+
+      - name: Upload changed files to the release
```

Comment:
Yes, this will replace the old file and that is done with this line of code here ` gh release upload $latest_tag $file --clobber`. This is done because of `--clobber`, it overwrites the existing files.